### ISSUE TITLE
NVDA doesn't read texts on hover when the element has a `transform: translate3d` property

### ISSUE THREAD

**Issue by wujekbogdan** [nvaccess/nvda#10762]
I was testing the [Swiper slider](https://swiperjs.com/) accessibility on various browsers with NVDA screen reader. It works well everywhere except for Firefox.

By default, the NVDA reader reads all texts on hover. It works for all texts except for texts that are inside slides. I found that the issue is caused by inline styles for the `.swiper-wrapper` element. The problem is caused by the `transform` property. If I disable that property in Firefox debugger then texts are read by NVDA.

### Steps to reproduce:

- Use Firefox 71.0 on Windows
- Go to this site: https://swiperjs.com/demos/020-navigation.html
- Switch slides and hover over the texts
- NVDA doesn't read slides' content

### Actual behavior:

- NVDA doesn't read slides' content.

### Expected behavior:

- NVDA should read slides' text on hover.

#### NVDA installed/portable/running from source:

installed

#### NVDA version:

2019.2.1

#### Windows version:

7 64-bit

#### Name and version of other software in use when reproducing the issue:

Firefox 71.0

#### Does the issue still occur after restarting your PC?

Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No.

**Comment by feerrenrut** [comment]
If this worked with Chrome + NVDA then it is almost certainly an issue in Firefox. Could you raise an issue and report the issue number here? https://bugzilla.mozilla.org/home

**Comment by wujekbogdan** [comment]
Done: https://bugzilla.mozilla.org/show_bug.cgi?id=1613626

**Comment by jcsteh** [comment]
This appears to be fixed, likely as part of the Cache the World project released in Firefox 113.
